forem/forem
# Fix more button not working after navigating back to the home page

## What type of PR is this? (check all applicable)

## Description
This PR fixes an issue where the "More" button on the homepage doesn't have a click event listener attached to it when you navigate from the homepage, to another page, and back to the homepage.

https://user-images.githubusercontent.com/17884966/107254949-cb1d2e00-6a05-11eb-93ee-e7ea319198bf.mov


However, as you can see in the video, a new issue happens where the opened "More" links remain opened after the first navigation. This is probably not _that_ big of an issue so I decided to leave it like this, rather than not fix the current bug.

## Related Tickets & Documents
Resolves #11897 

## QA Instructions, Screenshots, Recordings
1. Make sure you have at least 6 navigation links by visiting `/admin/navigation_links`
2. Visit the home page
3. Click "More"
4. Visit another page
5. Navigate to the homepage again by clicking the home logo link
6. Click "More"
7. See that it works
8. Repeat steps 4 - 6 and see that the menu links are still opened

### UI accessibility concerns?
Nope

## Added tests?

## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
Nope

## [optional] What gif best describes this PR or how it makes you feel?

![A cat plays whack a mole with a human using their finger as the mole](https://media.giphy.com/media/FmNXeuoadNTpe/giphy-downsized-large.gif)
